docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,467 @@
+  - within or outside of the challenge period for the specified DKG result (if
+  outside, then invalid) +
+  - have all members who signed a DKG result been selected by the sortition pool
+  to be in the group (if yes, then challenge invalid).
+
+| Group creation - DKG challenge
+| If the challenge notification was received within the challenge period and was
+  deemed justified, then: +
+  - the malicious DKG result should be immediately discarded (in the same
+  transaction in which notification happened) +
+  - all sortition pool members who signed the result should be slashed (fixed
+  amount) (in the same tx) +
+  - notifier should receive 5% from the total slashed amount (in the same tx) +
+  - the members of the signing group should be given another chance to publish
+  the DKG result
+  - DKG timeout timer and the result subission eligibility order shoul be reset.
+
+| Group creation - DKG challenge
+| If the challenge notification was received within the challenge period and was
+  not justified, then: +
+  - challenge transaction is reverted.
+
+| Group creation - DKG challenge
+| If the challenge notification (justified or not) was received outside of the
+  challenge period, then: +
+  - challenge transaction is reverted.
+
+| Group creation - DKG acceptance
+| Anyone can send a transaction requesting unlocking of the sortition pool and
+  requesting marking of the DKG result as accepted.
+
+| Group creation - DKG acceptance
+| Upon processing the transaction unlocking the sortition pool and marking the
+  DKG result as accepted, the random beacon contract should determine validity
+  of the transaction.
+
+| Group creation - DKG acceptance
+| When determining validity of the tx unlocking the sortition pool and accepting
+  the DKG result, the following aspects should be taken into account: +
+  - is the sortition pool locked? +
+  - has the challenge period already passed? +
+  Only if all the above conditions have been met, the tx is considered valid.
+
+| Group creation - DKG acceptance
+| If the transaction unlocking the sortition pool and marking the DKG result as
+  accepted gets approved, then: +
+  - the sortition pool should get unlocked +
+  - the DKG result should be accepted and a group should be created based on the
+  candidate group +  
+  - submitter should receive the fixed reward (in T), paid from the maintenance
+  pool to the submitter’s address (in the same tx) +
+// TODO: revisit 2 weeks removal time
+  - operators who became eligible for submitting the DKG result before the
+  member who submitted the DKG result should be punished by being removed from
+  the sortition pool for a fixed amount of time (2 weeks) (in the same tx).
+
+| Group creation - DKG timeout
+| The DKG submission timeout equals the group size multiplied by the number of
+  blocks for a member to become eligible to submit the DKG result. The timer
+  starts at the moment when the first member becomes eligible (the moment when
```

Comment:
We should include the off-chain DKG time. It's currently `72` blocks.

Reply:
Fixed, see 68f430e5fcb4f81e65aa832ca7c79b8f3774d297. I did not provide the exact value, just added the info that the added time should cover for the time of DKG generation.
Btw, is this `72 blocks` a governable param? If so, I will need to add another UAC for that.

Reply:
Nope, it's not. It's a fixed off-chain duration of the DKG protocol.